### PR TITLE
Update twitter.py

### PR DESCRIPTION
**Pull request by hellvix**
Applications submitted to Twitter's privileged list can now obtain user e-mail address. Fix for #825

### CODE REVIEW

**Comment by gustavi** [comment]
up

File: social/backends/twitter.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def get_user_details(self, response):
         """Return user details from Twitter account"""
         fullname, first_name, last_name = self.get_user_names(response['name'])
         return {'username': response['screen_name'],
-                'email': '',  # not supplied
+                'email': response.get('email'),
```
Comment (yprez) [inline social/backends/twitter.py]: I think it should be `response.get('email','')` for backwards compatibility (empty string and not None when email is missing).

Comment (hellvix) [inline social/backends/twitter.py]: You are right.

**Comment by omab** [comment]
I've merged this manually and moved the querystring parameter in the URL to the `params` option in `get_json`.

Thanks!

**Pull request closed without merging**